package.json

```diff
@@ -103,6 +103,7 @@
     "@types/express": "^4.17.14",
     "@types/fs-extra": "^9.0.13",
     "@types/jquery": "^3.5.10",
+    "@types/mocha": "^10.0.1",
```

Comment:
that's a _way_ newer version than the mocha we're using, but I guess it works ok anyway: our mocha is stuck in the past because of the ESM issue.

Reply:
Yeah, `@types/xyz` versions don't always correspond to `xyz`'s repositories so I wasn't sure what to do. At least for the file I converted the types seemed fine. I can try throwing a 8.x.x version on and see.